src/Core/SettingsChangesHistory.cpp

```diff
@@ -50,6 +50,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"query_plan_optimize_join_order_limit", 1, 1, "New setting"},
             {"query_plan_display_internal_aliases", false, false, "New setting"},
             {"allow_experimental_delta_lake_writes", false, false, "New setting."},
+            {"optimize_rewrite_like_to_range", false, true, "New setting"},
```

Comment:
If the default value is true this shous be `true, true` for new settings or CI will complain